lib/msf/core/payload/pingback/options.rb

```diff
@@ -0,0 +1,23 @@
+# -*- coding => binary -*-
+
+require 'msf/core'
+require 'msf/core/payload/pingback'
+
+#
+# This module provides datastore option definitions and helper methods for payload modules that support UUIDs
+#
+module Msf::Payload::Pingback::Options
+
+  def initialize(info = {})
+    super
+    register_advanced_options(
+      [
+        Msf::OptInt.new('PingbackRetries', [true, "How many additional successful pingbacks", 0]),
+        Msf::OptInt.new('PingbackSleep', [true, "Time (in seconds) to sleep between pingbacks", 30]),
+        Msf::OptString.new('PingbackUUID', [ false, 'A pingback UUID to use']),
+        Msf::OptBool.new('PingbackUUIDDatabase', [ true, 'Save the pingback UUID to the database', false]),
```

Comment:
I don't see the `PingbackUUID` or `PingbackUUIDDatabase` used anywhere.

Reply:
Fair.  I have no idea on the PingbackUUIDDatabase, but I still want to allow users to specify a UUID for the listener or the payload, but I have not implemented it.

Reply:
That's fine, we can make the session pickier in a later PR, but I don't think should add options before they're hooked up to something.

Reply:
Removed and pushed.